coderd/telemetry/telemetry.go

```diff
@@ -698,27 +699,46 @@ func ConvertWorkspaceProxy(proxy database.WorkspaceProxy) WorkspaceProxy {
 	}
 }
 
+func ConvertExternalProvisioner(id uuid.UUID, tags map[string]string, provisioners []database.ProvisionerType) ExternalProvisioner {
+	hashedTags := make(map[string]string, len(tags))
+	for k, v := range tags {
+		hash := sha256.Sum256([]byte(v))
+		hashedTags[k] = hex.EncodeToString(hash[:])
```

Comment:
we don't obscure workspace or template names, so I don't think we should obscure these.